woocommerce/woocommerce
# Remove update store details note

### All Submissions:






### Changes proposed in this Pull Request:

Remove the update store details note

Closes #34930 .






### How to test the changes in this Pull Request:



1. On a new site, do not complete the onboarding profiler and do not check the checkbox for "Setup for client" during onboarding
1. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
1. Under Tools -> Cron Events, run wc_admin_daily
1. Make sure the note with a title of "Edit your store details if you need to" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
